shuffle/cli/tests/shuffle-forge-cli-tests.rs

```diff
@@ -0,0 +1,282 @@
+        assert_eq!(output.status, ExitStatus::from_raw(0));
+        Ok(())
+    }
+}
+
+pub struct TransactionsWithNetworkAndRawFlag;
+
+impl Test for TransactionsWithNetworkAndRawFlag {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-with-networks-and-raw-flag"
+    }
+}
+
+impl AdminTest for TransactionsWithNetworkAndRawFlag {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let temp_dir = tempdir()?;
+        let base_path = temp_dir.path();
+        let helper = bootstrap_shuffle(ctx, base_path)?;
+        let base_path_string = base_path.to_string_lossy().to_string();
+        helper.overwrite_networks_config_into_toml(helper.networks_config().clone())?;
+        let output = std::process::Command::new(BINARY)
+            .args([
+                HOME_BASE_PATH,
+                base_path_string.as_str(),
+                "transactions",
+                "--raw",
+                "--network",
+                "forge",
+            ])
+            .output()?;
+        let std_out = String::from_utf8(output.stdout)?;
+        assert_modules_appear_in_txns(std_out.as_str(), true);
+        assert_eq!(output.status, ExitStatus::from_raw(0));
+        Ok(())
+    }
+}
+
+pub struct TransactionsWithNetworkRawAddressFlags;
+
+impl Test for TransactionsWithNetworkRawAddressFlags {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-with-networks-raw-and-address-flags"
+    }
+}
+
+impl AdminTest for TransactionsWithNetworkRawAddressFlags {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        let temp_dir = tempdir()?;
+        let base_path = temp_dir.path();
+        let helper = bootstrap_shuffle(ctx, base_path)?;
+        let base_path_string = base_path.to_string_lossy().to_string();
+
+        helper.overwrite_networks_config_into_toml(helper.networks_config().clone())?;
+        let output = std::process::Command::new(BINARY)
+            .args([
+                HOME_BASE_PATH,
+                base_path_string.as_str(),
+                "transactions",
+                "--raw",
+                "--network",
+                "forge",
+                "--address",
+                "24163AFCC6E33B0A9473852E18327FA9",
+            ])
+            .output()?;
+        let std_out = String::from_utf8(output.stdout)?;
+        assert_modules_appear_in_txns(std_out.as_str(), true);
+        assert_eq!(output.status, ExitStatus::from_raw(0));
+        Ok(())
+    }
+}
+
+pub struct TransactionsWithNetworkTailFlags;
+
+impl Test for TransactionsWithNetworkTailFlags {
+    fn name(&self) -> &'static str {
+        "shuffle::transactions-with-networks-and-tail-flags"
+    }
+}
+
+impl AdminTest for TransactionsWithNetworkTailFlags {
+    fn run<'t>(&self, ctx: &mut AdminContext<'t>) -> Result<()> {
+        // todo: add this test after we support random account creation.
+        // let temp_dir = tempdir()?;
```

Comment:
Please delete all of these comments and maybe even tests before merging. No dead code